srtcore/core.h

```diff
@@ -531,6 +688,18 @@ class CUDTGroup
     SRTU_PROPERTY_RW_CHAIN(CUDTGroup, int64_t,        latency,              m_iTsbPdDelay_us);
 };
 
+inline void srt_fwd_swap(CUDTGroup::BufferedMessage& a, CUDTGroup::BufferedMessage& b)
+{
+    a.swap_with(b);
+}
+
+namespace std
+{
+    template<>
+    inline void swap(CUDTGroup::BufferedMessage& a, CUDTGroup::BufferedMessage& b)
+    { ::srt_fwd_swap(a, b); }
+}
```

Comment:
What is the reason for namespace `std'?

Reply:
Because it provides an extra overload for `std::swap` (used by templates from the standard library), not adds a `swap` function.

Reply:
This can be seen from the code. But is there a reason to put that `swap` overload to the `std` namespace? Do you make use of it somehow, like ADL of std containers?

Reply:
Ok, I checked by blocking this version - looks like it's not in use now. I can remove it.